examples/hosts/app-integration/external-data/tests/index.tsx

```diff
@@ -31,26 +31,31 @@ export async function createContainerAndRenderInElement(element: HTMLDivElement)
 		const createResponse = await sessionStorageModelLoader.createDetached("1.0");
 		model = createResponse.model;
 
-		// Add a test task so we can see something.
-		model.taskList.addDraftTask("1", "testName", 3);
-
 		id = await createResponse.attach();
 	} else {
 		id = location.hash.slice(1);
 		model = await sessionStorageModelLoader.loadExisting(id);
 	}
 
-	// update the browser URL and the window title with the actual container ID
-	// eslint-disable-next-line require-atomic-updates
-	location.hash = id;
-	document.title = id;
+	model.baseDocument.addTaskList({ externalTaskListId: "task-list-test" });
```

Comment:
I see this is the app we use for tests, not the actual example app; this doesn't cause any issues there? For the case of several clients sharing a single doc, like we discussed during our call earlier.

Reply:
Ah forgot to update.